website/docusaurus.config.js

```diff
@@ -264,9 +264,7 @@ const config = {
       return {
         name: 'docusaurus-tailwindcss',
         configurePostCss(postcssOptions) {
-          // Appends TailwindCSS and AutoPrefixer.
-          postcssOptions.plugins.push(require('tailwindcss'));
-          postcssOptions.plugins.push(require('autoprefixer'));
+          postcssOptions.plugins.push(require("@tailwindcss/postcss"));
```

Comment:
As the original error suggests, the PostCSS plugin has moved to a separate package `@tailwindcss/postcss`. Autoprefixer is not needed anymore with Tailwind 4.

https://tailwindcss.com/docs/upgrade-guide#using-postcss